GitUI/CommandsDialogs/BrowseDialog/DashboardControl/UserRepositoriesList.Designer.cs

```diff
@@ -136,16 +128,64 @@ private void InitializeComponent()
             this.lblRecentRepositories.Margin = new System.Windows.Forms.Padding(0);
             this.lblRecentRepositories.Name = "lblRecentRepositories";
             this.lblRecentRepositories.Size = new System.Drawing.Size(411, 32);
-            this.lblRecentRepositories.TabIndex = 1;
+            this.lblRecentRepositories.TabIndex = 2;
```

Comment:
Please check that the tab order is logical

Reply:
Yep, the default focus is on the search box, after that comes the recent repo list, then the left menu. The overall behavior stays the same